module/VuFind/src/VuFind/ILS/Driver/PAIA.php

```diff
@@ -1864,6 +1864,13 @@ protected function paiaLogin($username, $password)
                 self::SCOPE_WRITE_ITEMS . ' ' .
                 self::SCOPE_CHANGE_PASSWORD;
 
+        // append additional scopes via config
+        if (!empty($this->config['PAIA']['additionalScopes'])) {
+            foreach ($this->config['PAIA']['additionalScopes'] as $scope) {
+                $post_data['scope'] .= ' ' . constant("self::$scope");
+            }
+        }
```

Comment:
Would it make sense to simplify with null coalescing, like:

```suggestion
        foreach ((array)($this->config['PAIA']['additionalScopes'] ?? []) as $scope) {
            $post_data['scope'] .= ' ' . constant("self::$scope");
        }
```

? This should also correctly cover the edge case where somebody sets a single `additionalScopes` value without including the array brackets.

Reply:
Thanks for the suggestion.